### ISSUE TITLE
GetRows returns empty list for non-empty sheet

### ISSUE THREAD

**Issue by bubenkoff** [Luxurioust/excelize#39]
Thanks for this project!

```
xls, err := excelize.OpenFile("data/test2.xlsx")
log.Println(xls.GetRows("Sheet1"))
```

Gets me an empty list while the file is not empty

https://www.dropbox.com/s/c4b0piomygv6n2g/test2.xlsx?dl=0

**Comment by Luxurioust** [comment]
@bubenkoff Thanks for your issue. This library needs Go version 1.8 or later after fixed issue #32 for collecting XML all attributes. Relate issue #35.

**Comment by bubenkoff** [comment]
thank you very much - works!
